legend-engine-xt-serviceStore-grammar/src/test/java/org/finos/legend/engine/language/pure/grammar/test/TestServiceStoreMappingGrammarRoundtrip.java

```diff
@@ -161,64 +152,64 @@ public void testSimpleServiceStoreMappings()
     public void testServiceStoreWithMappingTestSuites()
```

Comment:
Can we please have a testSuite with multiple tests? I feel that will uncover a lot issues

Reply:
added in test runnger in the service store module

Reply:
I meant a test to cover roundtrip grammar here

Reply:
MappingTestRunner covers more than the roundtrip. Not sure what you expect to uncover
" I feel that will uncover a lot issues"
But i updated the tests